javascript/javascript2/week3/The fastest presser in this realm.js

```diff
@@ -0,0 +1,51 @@
+"use strict";
+
+// counter for L
+let totalL = 0;
+const elL = document.querySelector("#countL");
```

Comment:
Try and avoid acronyms. Instead of btn just write button instead of el just write element